apps/yapms/src/routes/app/+layout.svelte

```diff
@@ -32,6 +33,8 @@
 		if (event.code === 'KeyR' && event.target instanceof Element && event.target.id === 'map-div') {
 			reapplyPanZoom();
 		}
+
+		handleCandidateSelectionShortcut(event.code);
```

Comment:
This should share the condition reapplyPanZoom() has that checks if the target is 'map-div'. Otherwise, pressing number keys anywhere on the page including while typing in an input will result in a candidate switch.

Reply:
Should be resolved now.